src/TRestGeant4Metadata.cxx

```diff
@@ -1457,10 +1457,10 @@ void TRestGeant4Metadata::PrintMetadata() {
     }
     RESTMetadata << "Number of active volumes: " << GetNumberOfActiveVolumes() << RESTendl;
     for (unsigned int n = 0; n < GetNumberOfActiveVolumes(); n++) {
-        RESTMetadata << "Name: " << GetActiveVolumeName(n)
-                     << ", ID: " << GetActiveVolumeID(GetActiveVolumeName(n))
-                     << ", maxStep: " << GetMaxStepSize(GetActiveVolumeName(n)) << "mm "
-                     << ", chance: " << GetStorageChance(GetActiveVolumeName(n)) << RESTendl;
+        const auto name = GetActiveVolumeName(n);
+        RESTMetadata << "Name: " << name << ", ID: " << fGeant4GeometryInfo.GetIDFromVolume(name)
```

Comment:
Would not be better to re-implement the method `GetActiveVolumeID`?

We could just:

```
Int_t GetActiveVolumeID( name ) { return fGeant4GeometryInfo.GetIDFromVolume(name); }
```

Reply:
I'm not sure it won't cause problems elsewhere but it could be done. But in that case why not just implement `TRestGeant4Metadata::GetVolumeID(const std::string& name)`? The idea of the `TRestGeant4GeometryInfo` is to not overload `TRestGeant4Metadata` with members and methods related to geometry (same things for `TRestGeant4PhysicsInfo`, but I guess some of them could be duplicated, if it's deemed useful.